.gitignore

```diff
@@ -10,3 +10,5 @@ cached_data
 */out/*
 !*/out/README.md
 .remake
+.httr-oauth
+lib
```

Comment:
are you sure you want to ignore lib? it might contain scripts shared across several data analysis phases, not just user-specific config info (and i think it should be possible to share config info among users for google drive, too, if you use an ID rather than a relative path to the folder (as you're doing))